pkg/operator/controller/status_test.go

```diff
@@ -7,15 +7,14 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 	operatorv1 "github.com/openshift/api/operator/v1"
 
-	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func TestComputeStatusConditions(t *testing.T) {
+func TestComputeOperatorStatusConditions(t *testing.T) {
 	type testInputs struct {
-		haveNamespace                           bool
-		numWanted, numAvailable, numUnavailable int
+		haveNamespace           bool
+		numWanted, numAvailable int
```

Comment:
It feels like we have lost some test coverage here, because whereas before we were counting things that had a positive number in some particular field, now we are counting things that have a slice of things where one of those things has a specific value in one particular field and another specific value in another particular field.  That is to say, there are many more ways the logic could go wrong.  What do you think about adding inputs for the number of dnses without an `Available` condition, number with `Available` set to `Unknown`, and number with `Available` set to `False`, as well as a test case for each of these and for a combination of several?

Reply:
Thank you for the review. I have addressed all your comments in my latest commit, except this one. My understanding is `numUnavailable` was removed because the `Progressing` and `Available` status conditions are now based on the `DNS` status condition instead of comparing the number of available and unavailable `DaemonSets`.